ignite/pkg/cosmosgen/generate.go

```diff
@@ -104,23 +117,141 @@ func (g *generator) setup() (err error) {
 			if err != nil {
 				return err
 			}
+			includes := []string{}
```

Comment:
we should avoid literals for empty slices

```suggestion
			includes := make([]string, 0)
```

Reply:
Already changed to in #3696 to:

```go
var includes []string
```